docs/sources/best-practices/best-practices-for-creating-dashboards.md

```diff
@@ -0,0 +1,55 @@
++++
+title = "Best practices for creating dashboards"
+description = "Best practices for creating dashboards in Grafana"
+type = "docs"
+[menu.docs]
+weight = 200
++++
+
+# Best practices for creating dashboards
+
+This page outlines some best practices to follow when creating Grafana dashboards.
+
+## Before you begin
+
+Here are some principles to consider before you create a dashboard.
+
+### A dashboard should tell a story or answer a question
+
+What story are you trying to tell with your dashboard? Try to create a logical progression of data, such as large to small or general to specific. What is the goal for this dashboard? (Hint: If the dashboard doesn't have a goal, then ask yourself if you really need the dashboard.)
+
+Keep your graphs simple and focused on answering the question that you are asking. For example, if your question is "which servers are in trouble?", then maybe you don't need to show all the server data. Just show data for the ones in trouble.
+
+### Dashboards should reduce cognitive load, not add to it
+
+_Cognitive load_ is basically how hard you need to think about something in order to figure it out. Make your dashboard easy to interpret. Other users and future you (when you're trying to figure out what broke at 2AM) will appreciate it.
+
+Ask yourself:
+- Can I tell what exactly each graph represents? Is it obvious, or do I have to think about it?
+- If I show this to someone else, how long will it take them to figure it out? Will they get lost?
+
+### Have a monitoring strategy
+
+It's easy to make new dashboards. It's harder to optimize dashboard creation and adhere to a plan, but it's worth it. This strategy should govern both your overall dashboard scheme and enforce consistency in individual dashboard design.
+
+Refer to [Common observability strategies]({{< relref "common-observability-strategies.md" >}}) and [Dashboard management maturity levels]({{< relref "dashboard-management-maturity-levels.md" >}}) for more information.
+
+### Write it down
+
+Once you have a strategy or design guidelines, write them down to help maintain consistency over time. Check out this [Wikimedia runbook example](https://wikitech.wikimedia.org/wiki/Performance/Runbook/Grafana_best_practices).
+
+## Best practices to follow
+
+- When creating a new dashboard, make sure it has a meaningful name.
+  - If you are creating a dashboard to play or experiment, then put the word `TEST` or `TMP` in the name.
+  - Consider including your name or initials in the dashboard name so that people know who owns the dashboard.
```

Comment:
What about putting such information into the tags?

Reply:
I'm in favor of putting information like this as tags. `TEST` or `TMP` doesn't describe the purpose of the dashboard, but I'd like to be able to quickly filter them out. 

In the case of ownership, I've found that creating folder for individual team members can help here. Sometimes you just want to experiment with new dashboards, and giving people their own "sandbox" folder makes it clear who owns it. I also don't believe that you should link production dashboards to individual team members, i.e. personal folder should only ever contain experiments/tests/temporary dashboards. Once a dashboard is ready for prime time, it should be moved into the proper folder.

Reply:
I like this idea, and I know that we have at least some really big installations that does it that way. I still think, for sorting reasons, that prefixing with something that groups your dashboards together makes sense — but it makes sense to use tags for this at the moment.

For the future, if Grafana had personal/team dashboards and user profiles as first-class citizens that would solve the experiment/testing/temporary use-case (and potentially eliminate the need for setting up a second Grafana for the dashboards that make it into the mainline that @davkal mentioned in https://github.com/grafana/grafana/pull/27057#discussion_r476640826 ).

I couldn't find it now, but I've seen some discussion somewhere around having label-tags with key-value separation, which would be another way of adding something like `owner:emil` without having to clutter the tags UX with a lot of tags that are really drill downs from a general tag category. That would help removing some of the arguments against recommending tags as a best practice for say dashboard ownership.

Reply:
That might work if there aren't too many users or dashboard creators.